website/docs/r/video_analyzer.html.markdown

```diff
@@ -0,0 +1,132 @@
+---
+subcategory: "Video Analyzer"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_video_analyzer"
+description: |-
+  Manages a Video Analyzer.
+---
+
+# azurerm_video_analyzer
+
+Manages a Video Analyzer.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "video-analyzer-resources"
+  location = "West Europe"
+}
+
+resource "azurerm_storage_account" "example" {
+  name                     = "examplestoracc"
+  resource_group_name      = azurerm_resource_group.example.name
+  location                 = azurerm_resource_group.example.location
+  account_tier             = "Standard"
+  account_replication_type = "GRS"
+}
+
+resource "azurerm_user_assigned_identity" "example" {
+  name                = "exampleidentity"
+  resource_group_name = azurerm_resource_group.example.name
+  location            = azurerm_resource_group.example.location
+}
+
+resource "azurerm_role_assignment" "contributor" {
+  scope                = azurerm_storage_account.example.id
+  role_definition_name = "Storage Blob Data Contributor"
+  principal_id         = azurerm_user_assigned_identity.example.principal_id
+}
+
+resource "azurerm_role_assignment" "reader" {
+  scope                = azurerm_storage_account.example.id
+  role_definition_name = "Reader"
+  principal_id         = azurerm_user_assigned_identity.example.principal_id
+}
+
+resource "azurerm_video_analyzer" "example" {
+  name                = "exampleanalyzer"
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+
+  storage_account {
+    id          = azurerm_storage_account.example.id
+    identity_id = azurerm_user_assigned_identity.example.id
+  }
+
+  identity {
+    type = "UserAssigned"
+    identity_ids = [
+      azurerm_user_assigned_identity.example.id
+    ]
+  }
+
+  tags = {
+    environment = "staging"
+  }
+
+  depends_on = [
+    azurerm_user_assigned_identity.example,
```

Comment:
this should be implied from the two role assignments, which depend on this?

```suggestion
```

Reply:
Great catch